src/client/src/utils/constants.ts

```diff
@@ -40,6 +40,14 @@ const ARIA_LABELS_NAVIGATION = defineMessages({
   }
 });
 
+const PAGEID = {
```

Comment:
Curious why don't we use string enum for this? Is there an advantage in terms of readability/type safety to do it this way rather than with enum strings?

Reply:
Good point - it is better to use enum since these are ranged values. I just updated the PR to make it an enum instead of using a plain object

Reply:
Oh this update works too. I was thinking more like
```ts
enum SomeEnum {
  SomeVal="SomeString",
  SomeVal2="SomeString2"
}
```
Makes your code simple again while providing type safety.